subworkflows/ebi-metagenomics/rrna_extraction/main.nf

```diff
@@ -0,0 +1,44 @@
+
+include { INFERNAL_CMSEARCH           } from '../../../modules/ebi-metagenomics/infernal/cmsearch/main'
+include { CMSEARCHTBLOUTDEOVERLAP     } from '../../../modules/ebi-metagenomics/cmsearchtbloutdeoverlap/main'
+include { EASEL_ESLSFETCH             } from '../../../modules/ebi-metagenomics/easel/eslsfetch/main'
+
+workflow RRNA_EXTRACTION {
+
+    take:
+    ch_fasta     // channel: [ val(meta), [ fasta ] ]
+    val_rfam     // value: /path/to/rfam for cmsearch
+    val_claninfo // value: /path/to/claninfo for cmsearchtbloutdeoverlap
+
+    main:
+
+    ch_versions = Channel.empty()
+
+    INFERNAL_CMSEARCH(
+        ch_fasta,
+        file(val_rfam)
```

Comment:
Do we need to convert them to "file"? Why don't we just ask users to send a "file"?. If would be easier to catch if the file doesn't exist for example

Reply:
yes that makes sense! I'll change this and convert them into files in the unit test as a good example